formspree/templates/forms/list.html

```diff
@@ -35,6 +35,9 @@
           <a href="{{ url_for('form-submissions', hashid=form.hashid) }}">{{ form.counter }} submissions</a>
         {% endif %}
       </td>
+		<td>
+		<a href="{{ url_for('form-deletion', hashid=form.hashid) }}">Delete</a>
+		</td>
```

Comment:
Since we can't use `GET` endpoints, I think the simplest solution is to use a `POST` endpoint and replace this `<a>` with an inline form with just a button -- I think this is also vulnerable, but a little less (and I think it is an approach being used in other parts of Formspree).